soap/soap_constants.bal

```diff
@@ -15,10 +15,10 @@
 // under the License.
 
 // SOAP related constants
-@final string SOAP11_NAMESPACE ="http://schemas.xmlsoap.org/soap/envelope/";
-@final string SOAP12_NAMESPACE ="http://www.w3.org/2003/05/soap-envelope";
-@final string SOAP11_ENCODING_STYLE ="http://schemas.xmlsoap.org/soap/encoding/";
-@final string SOAP12_ENCODING_STYLE ="http://www.w3.org/2003/05/soap-encoding";
+final string SOAP11_NAMESPACE ="http://schemas.xmlsoap.org/soap/envelope/";
+final string SOAP12_NAMESPACE ="http://www.w3.org/2003/05/soap-envelope";
+final string SOAP11_ENCODING_STYLE ="http://schemas.xmlsoap.org/soap/encoding/";
+final string SOAP12_ENCODING_STYLE ="http://www.w3.org/2003/05/soap-encoding";
 
-// Symbols
-@final string EMPTY_STRING ="";
+// Error Codes
+final string SOAP_ERROR_CODE = "(wso2/soap)SoapError";
```

Comment:
Shall we use square bracket {} on error code as a convention and not () : {wso2/soap}SOAPError

Reply:
Fixed with https://github.com/wso2-ballerina/module-soap/pull/30